alicloud/data_source_alicloud_oss_buckets.go

```
@@ -365,9 +367,8 @@ func bucketsDescriptionAttributes(d *schema.ResourceData, buckets []oss.BucketPr
 		raw, err = client.WithOssClient(func(ossClient *oss.Client) (interface{}, error) {
 			return ossClient.GetBucketWebsite(bucket.Name)
 		})
-		if err != nil && !IsExceptedErrors(err, []string{NoSuchWebsiteConfiguration}) {
```

Why ignore the error?